src/applications/edu-benefits/10282/pages/applicantInformation/applicantDescription.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+const labels = {
+  veteran: "I'm a Veteran",
+  veteranSpouse: "I'm a Veteran's Spouse",
+  veteranChild: "I'm a Veteran's Child",
+  veteranCaregiver:
+    "I'm a Veteran's Caregiver (who isn't their spouse or child)",
```

Comment:
`spouse`, `child`, and `caregiver` should all be lowercase here

Reply:
done